app/models/manageiq/providers/ovirt4/infra_manager.rb

```diff
@@ -0,0 +1,29 @@
+require 'ovirtsdk4'
+
+class ManageIQ::Providers::Ovirt4::InfraManager < ManageIQ::Providers::InfraManager
```

Comment:
@jhernand sorry to increase the friction on this. I have no objection on renaming the provider, but I think the path you lay out here and the one that is _sneak peaked_ in the slide you link are a good way to go. But maybe I'm not getting the whole picture.

So yeah, to get the ovirt4 sdk in as fast as possible, use the toggle approach